service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -182,6 +183,7 @@ public String startLandingZoneResourceCreationJob(
       String resultPath) {
     var jobDescription =
         "Inner flight to create landing zone resources. definition='%s', version='%s'";
+    MetricUtils.incrementLandingZoneCreation(landingZoneRequest.definition());
```

Comment:
This metric could be clearer -- it's not measuring the creation of a landing zone, it's measuring the _start_ of the creation.

Reply:
This counts number of lz create requests.